public/javascripts/controllers/AuthCtrl.js

```diff
@@ -55,4 +55,27 @@ function($scope, $state, auth){
 
   $scope.speciality = "Psychiatry";
 //_______________________________________
+
+
+$scope.signinCallback =  function(authResult) {
```

Comment:
Where is this method used ?
This method is repeated in PublicScript.js also. Right?